recipe/meta.yaml

```diff
@@ -21,14 +24,13 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }}                 # [cuda_compiler_version not in (undefined, 'None')]
-    - sysroot_linux-64 ==2.17                # [linux64]
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - pytorch                                # [build_platform != target_platform]
-    - pytorch =*={{ torch_proc_type }}*      # [build_platform != target_platform]
+    - {{ compiler('cuda') }}                             # [cuda_compiler_version not in (undefined, 'None')]
+    - sysroot_linux-64 ==2.17                            # [linux64]
+    - python                                             # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}                 # [build_platform != target_platform]
     - cmake
-    - ninja
+    - ninja        # [unix]
+    - llvm-openmp  # [osx]
     - llvm-openmp  # [osx]
```

Comment:
Just noticed that this is now duplicated. That was most likely the fault of my suggestion, sorry.